### DISCUSSION TITLE
rancher password reset on live production system

### DISCUSSION THREAD

**Discussion by zarafree** [rancher/k3os#800] [General]
Hi All,

Need your help to reset the password of rancher user. One of team member forgotten password of rancher user of production system. I tried to reset password through grub procedure overriding k3os.password parameter but it doesn't help. Is there any other way to reset the password of rancher user in k3os?

**Comment by dweomer** [comment]
You'll need to run `sudo k3os config` to apply your changes.

**Comment by zarafree** [comment]
Thanks but I lost access to my system and rancher is only user which I had. I can not run above command as no access to the system. Can you please help me on this?

**Comment by dweomer** [comment]
If your press `e` during the grub boot screen you can edit the `k3os.password` boot parameter and then login via console, apply your configuration changes, and then reboot and all should be well.

If you do not have access to the system at boot time but you do have access to the cluster via kubeconfig you can spin up a privileged pod on the node with a host-path mount of `/` to `/host` inside the container and edit the config there.
